opta/core/aws.py

```diff
@@ -22,10 +23,10 @@ class AwsArn(TypedDict):
     resource_type: Optional[str]
 
 
-class AWS:
+class AWS(CloudClient):
     def __init__(self, layer: "Layer"):
-        self.layer = layer
         self.region = layer.root().providers["aws"]["region"]
+        super(AWS, self).__init__(layer)
```

Comment:
```suggestion
        super().__init__(layer)
```

We should use Python 3's simplified form of `super` which doesn't require passing in the current class and `self`.